tox.ini

```diff
@@ -27,36 +28,36 @@ setenv =
     PYTHONPATH = {toxinidir}/tests:{toxinidir}
     PYTHONDEVMODE = 1
 
-deps =
-    flit>=3.9
+extras = testing
 
-    django3.2: Django>=3.2,<3.3
+deps =
     django4.2: Django>=4.2,<4.3
     django5.0: Django>=5.0,<5.1
-    djmain: git+https://github.com/django/django.git@main#egg=Django
```

Comment:
Do we no longer want to test against Django's main branch? Perhaps in an automated / allowed to fail pipeline somewhere?

Reply:
We never actually tested against Django main. That type of support is usually added to the next Wagtail release, so this was good to tidy